contracts/contract-config.js

```diff
@@ -101,4 +109,4 @@ module.exports = {
     userReplicaSetBootstrapAddress: '0x3d2563ACCD9E6D189bA2a61F116905D520054286',
     registryAddress: '0x793373aBF96583d5eb71a15d86fFE732CD04D452'
   }
-}
\ No newline at end of file
+}
```

Comment:
A general note about this file, it's modified during local-setup, but not excluded from the gitignore - but causes a diff. 
You don't need to change here, but would be nice if this file was gitignored and there was a template file for this that is copied over